### DISCUSSION TITLE
Is there a way to use internal module api (use --add-exports in compiler)

### DISCUSSION THREAD

**Discussion by hermannpencole** [jbangdev/jbang#982] [Q&A] [answered]
Hi,
My code needs to use --add-exports to compile in java.
How i can use it with Jbang ?
Thanks

**Comment by maxandersen** [comment, marked as answer]
but you can always use `//JAVAC_OPTIONS --add-exports xyz` to add compiler flags.

p.s. we haven't added support for module dependencies yet (no one really asked for it yet :), if you can give example of usecase you are after on #559 #618 that would be useful.
